static/installer.sh

```diff
@@ -276,24 +284,24 @@ install_yum_repo ()
   cat <<EOF > /etc/yum.repos.d/tarantool_${ver_repo}.repo
 [tarantool_${ver_repo}]
 name=${OS_NAME}-${dist} - Tarantool
-baseurl=https://download.tarantool.org/tarantool/${repo_path}${ver}/${OS_CODE}/${dist}/${ARCH}/
-gpgkey=https://download.tarantool.org/tarantool/${repo_path}${ver}/gpgkey
+baseurl=https://download.tarantool.org/tarantool/${repo_path}${ver}${gc64}/${OS_CODE}/${dist}/${arch}/
```

Comment:
It seems it shouldn't be changed.

Reply:
if we don't change the value of repo name, we will not be able to install gc64 version of Tarantool

Reply:
done